scripts/deploy.ts

```diff
@@ -29,20 +30,22 @@ async function main() {
 
   // Deploy proxy
   const Factory = await ethers.getContractFactory("TablelandTables");
-  const tables = await (
-    (await upgrades.deployProxy(Factory, [baseURI], {
-      kind: "uups",
-      timeout: pollTimeout,
-      pollingInterval: pollInterval,
-    })) as TablelandTables
-  ).deployed();
-  console.log("New proxy address:", tables.address);
+  // @ts-expect-error ignore `Conversion of type 'Contract'` error since
```

Comment:
i tried to get around this error but couldn't. the type that `deployProxy` _says_ it should return is an ethers `Contract`, but if you log it, it's a `BaseContract`—which is what `TablelandTables` extends. ignoring it resolved the issue.